tools/ctu-build-new/ctu-build.py

```diff
@@ -221,12 +237,22 @@ def create_external_fn_maps(ctuindir):
                 out_file.write('%s %s\n' % (mangled_name, ast_files.pop()))
 
 
-if not os.path.exists(mainargs.xtuindir):
-    os.makedirs(mainargs.xtuindir)
-clear_file(os.path.join(mainargs.xtuindir, 'cfg.txt'))
-clear_file(os.path.join(mainargs.xtuindir, 'definedFns.txt'))
-clear_file(os.path.join(mainargs.xtuindir, 'externalFns.txt'))
-clear_file(os.path.join(mainargs.xtuindir, 'externalFnMap.txt'))
+def create_external_fn_maps(ctuindir):
+    """Creates the function maping file for each triple."""
+
+    triple_paths = glob.glob(os.path.join(ctuindir, TEMP_EXTERNAL_FNMAP_FOLDER, '*'))
+    triple_dirs = filter(os.path.isdir, triple_paths)
+    triples = map(os.path.basename, triple_dirs)
+    for triple in triples:
+        create_external_fn_maps_for_triple(ctuindir, triple)
+
+
+if not os.path.exists(mainargs.ctuindir):
+    os.makedirs(mainargs.ctuindir)
+clear_file(os.path.join(mainargs.ctuindir, 'cfg.txt'))
```

Comment:
We no longer generate `cfg.txt`. I think the related code can be removed. Maybe there are other files we no longer generate from these ones.